cmds/storaged/main.go

```diff
@@ -65,6 +66,12 @@ func main() {
 		log.Info().Msg("shutting down")
 	})
 
+	go func() {
+		if err := http.ListenAndServe(":8081", http.DefaultServeMux); err != nil {
```

Comment:
Are you serious ? :'D please make this configurable via the command line, and make the default something with 5 figures. try 28682 (it's my birthday)